supervisor/supervisorctl.py

```diff
@@ -1053,7 +1053,7 @@ def _formatChanges(self, added_changed_dropped_tuple):
             for name in names:
                 self.ctl.output("%s: %s" % (name, changedict[name]))
         else:
-            self.ctl.output("No config updates to processes")
```

Comment:
"Process" in this context refers to a program running under supervisord.  This message uses the plural form "processes" because it refers to more than one.